### PR TITLE
Log in finished

### PR DESCRIPTION
**Pull request by arihilmir**
Now we can log in if our user exists in DB, 
Our users' password info is secured with hashing of the passwords
Db location updated (there were some problems with finding the path to DB -> moved to root of the project, it should cause fewer problems).

### CODE REVIEW

File: ViewModel/Authorisation.cs [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 namespace ViewModel
 {
 	public class Authorisation
 	{
-        readonly DataServices.Services.IPersonService service;
-        public Authorisation(DataServices.Services.IPersonService personService)
+        readonly IPersonService service;
+        public Authorisation(IPersonService personService)
         {
             Console.WriteLine($"{Directory.GetCurrentDirectory()}");
```
Comment (OlegAndrus12) [inline ViewModel/Authorisation.cs]: Console?

File: View/Widgets/LogInNavbar.xaml [hunk 2]
```diff
@@ -22,8 +22,15 @@
         </Grid.ColumnDefinitions>
         
         <local:logo x:Name="logo" Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2"  Width="180" Height="180" HorizontalAlignment="Left"/>
-        <TextBox HorizontalAlignment="Left" FontSize="16" Foreground="#959595" Height="30" Grid.Row="2" Grid.Column="0" Grid.ColumnSpan="2" Text="Username" Width="250" Margin="25 0 25 0"/>
-        <TextBox HorizontalAlignment="Left" FontSize="16" Foreground="#959595" Height="30" Grid.Row="3" Grid.Column="0" Grid.ColumnSpan="2" Text="Password" Width="250" Margin="25 0 25 0"/>
+        <StackPanel Orientation="Vertical" Grid.Row="2" Grid.Column="0" Grid.ColumnSpan="2" Margin="25 0 25 0">
```
Comment (OlegAndrus12) [inline View/Widgets/LogInNavbar.xaml]: As Nikita consider margins are the biggest possible evil

**Review by OlegAndrus12 — APPROVED** [review]

File: DataServices/SqliteDbContext.cs [hunk 3]
```diff
@@ -12,11 +12,12 @@ namespace DataServices
     public class SqliteDbContext : DbContext, IDataSource
     {
         private string _connectionString = null;
-        public SqliteDbContext(string connectionString = null)
+        public SqliteDbContext(string connectionStringPath = @"..\..\..\univerity-db.db")
         {
             var connectionStringBuilder = new SqliteConnectionStringBuilder();
             // TODO: create adequate location for DB.
-            connectionStringBuilder.DataSource = @"..\..\..\..\DataServices\univerity-db.db";
+            Console.WriteLine($"{Directory.GetCurrentDirectory()}");
```
Comment (OlegAndrus12) [inline DataServices/SqliteDbContext.cs]: console?

File: View/Widgets/LogInNavbar.xaml (hunk 2 above)
Comment (RomanLevkovych) [reply View/Widgets/LogInNavbar.xaml]: Maybe, but I took this from TextBox. Just copied layout.

Comment (Scaler358) [reply View/Widgets/LogInNavbar.xaml]: Margins for positions are bad. Margins for whitespace around element are good

File: DataServices/SqliteDbContext.cs (hunk 3 above)
Comment (Scaler358) [reply DataServices/SqliteDbContext.cs]: We should use Logger instead of console

**Pull request merged**